docs/src/main/sphinx/appendix/from-hive.rst

```diff
@@ -132,20 +132,3 @@ Trino query::
 
     date_diff('day', startdate, enddate)
 
-Overwriting data on insert
```

Comment:
Hmm,
I think i preferred old location as it's strict issue for users migrating from Hive. 
So maybe we could fix the description here and just link this location from Hive section ?
You could add anchor here and refer to it from hive.rst.

@mosabua - thoughts ?

Reply:
Sure. I will add an anchor in `hive.rst` and keep the original location. `Migrating from hive` is a great document but it is not referenced anywhere in hive connector documentation so it is difficult for users to navigate and land on this crucial information regarding insert overwrite. I think we should also add a reference to `migrating from hive` from `hive.rst`. 

How about this text at the very bottom of hive.rst?

> If you are migrating from hive to Trino, check [Migrating from Hive](https://trino.io/docs/current/appendix/from-hive.html)

OR 

We can move `migrate from hive` from appendix to  under hive connector like below:

![image](https://user-images.githubusercontent.com/14219201/144821294-74741a87-100a-4700-9da6-441fc5933c7d.png)


What do you think @mosabua @aczajkowski ?

Reply:
For me this looks good, but let's wait for @mosabua opinion.

Reply:
I would leave the file in the appendix since you could migrate from Hive to e.g. Delta Lake or Iceberg as well.. 

And adding a link to it sounds like a good idea. Maybe in the "SQL support" section in the Hive connector doc?

Reply:
Got it. I will keep the doc in the appendix and will add link from the hive.rst